### PR TITLE
Parse integer from TEST_INTENSITY bug fix

### PR DESCRIPTION
**Pull request by Ramakrish1994**

### CODE REVIEW

File: scripts/build-merge [hunk 1]
```diff
@@ -127,6 +127,11 @@ echo-do cp examples/movies/spark/target/scala-2.10/movies-spark.jar ${STAGE_MOVI
 echo-do mkdir -p stage/docs/scala/store
 echo-do mv doc/target/scala-2.11/unidoc stage/docs/scala/store/${version}
 
+#set NSEEDS
```
Comment (Topher-the-Geek) [inline scripts/build-merge]: This comment tells me what the line of code tells me.  Maybe change the comment to:
Number of seeds for psuedo-random tests.

**Comment by Topher-the-Geek** [comment]
This is on the right track. We need to remove intensity, and replace its uses with alternatives. See the [bug post](https://forum.treode.com/t/parse-integer-from-test-intensity/48) for details.

File: scripts/build-merge (hunk 1 above)
Comment (Ramakrish1994) [inline scripts/build-merge]: Hi Topher,

With regards to the below change, I also had to set NSEEDS in two other
scripts namely
-build-pull-request ( what comment do I add here?)
-build-periodic-test ( would number of seeds for periodic-test suffice for
this?)


I had one more doubt regarding removing intensity from
*disk/stub/com/treode/disk/stubs/CrashChecks.scala.*

There are actually more uses for ncrashes in this file, in the function
forAllCrashes(seed : Long). What do I do about these?

Thank you.

With Regards,
Sanjay

**Comment by Topher-the-Geek** [comment]
You can use the same comment for NSEEDS in the three build scripts.

As for the other uses of ncrashes in CrashChecks, that method can be removed. When you do so though, it'll break the loop I mentioned earlier. That loop needs to choose a random target, and then call forCrash directly. Take a look at how the to-be-removed forAllCrashes method chooses the target, and move that down into the to-be-kept forAllCrashes.

**Comment by Ramakrish1994** [comment]
forCrash is being called in a loop that runs either count or ncrashes times, but I see that the target is such that it always lies between 1 and count. So, am I supposed to call forCrash inside a loop in the loop discussed above or can I directly call it once with some random value between 1 and count? Further there is another variable count in to-be-retained for loop which seems to be keeping count of the number of times forCrash is called. Is that to incremented by 1 if the latter was true?

**Comment by Topher-the-Geek** [comment]
To facilitate this discussion, I want to use distinguished names for [forAllCrashes-seed](https://github.com/Treode/store/blob/69bb0258c25e20c8ed05a1f5e7f1e75b374f7790/disk/stub/com/treode/disk/stubs/CrashChecks.scala#L113) and [forAllCrashes](https://github.com/Treode/store/blob/69bb0258c25e20c8ed05a1f5e7f1e75b374f7790/disk/stub/com/treode/disk/stubs/CrashChecks.scala#L130).

Right now, it's like this:

    forAllCrashes-seed:
        val count = count steps of test when not crashing
        if (ncount > ncrashes)
          for (ncrashes times)
            choose a random point to crash between 1 and count
            test with crash at that step
          that's ncrashes tests
        else
           for (count times)
             test with crash at step i
           that's count tests

    forAllCrashes
       for (nseeds / ncrashes times)
         count += forAllCrashes-seed (choose a random seed)
       use count to compute average time

You need to effectively set ncrashes = 1. In fact, you could do exactly that, just set it to 1. I'm suggesting though, that when ncrashes=1 always, having the loop (of forAllCrashes-seed) within a loop of (forAllCrasehs) is pointless. Instead, just fold it all into one loop.

    forAllCrashes
      for (nseeds)
        choose a random seed
        val count = count steps of test when not crashing
        choose a random point to crash between 1 and count
        test with crash at that step
      use nseeds to compute average time

Hope that helps.

**Comment by Ramakrish1994** [comment]
Thanks, this helped a lot.

**Comment by Topher-the-Geek** [comment]
Great!

Sorry about this, but I completely missed something important the first time. In the scripts, `NSEEDS` needs to be set before the tests are run, and it needs to be exported, for example `export NSEEDS=10`.

**Pull request closed without merging**